e2e/client/playwright/workspace.spec.ts

```diff
@@ -0,0 +1,156 @@
+import {test, expect} from '@playwright/test';
+import {Monitoring} from './page-object-models/monitoring';
+import {restoreDatabaseSnapshot, s} from './utils';
+
+test.describe('Custom Workspace', async () => {
+    test.describe('Create custom workspace', async () => {
+        test('save custom workspace', async ({page}) => {
+            await restoreDatabaseSnapshot();
+            await page.goto('/#/workspace/monitoring');
+
+            await page.locator(s('monitoring--selected-desk')).click();
+            await page.locator(s('create-workspace')).click();
+            await expect(page.locator(s('workspace-modal', 'workspace-name'))).toBeVisible();
+            await expect(page.locator('button', {hasText: 'save'})).toBeDisabled();
+            await page.locator(s('workspace-modal')).locator(s('workspace-name')).fill('new custom workspace');
+            await expect(page.locator('button', {hasText: 'save'})).toBeEnabled();
+            await page.locator('button', {hasText: 'save'}).click();
+            await expect(page.locator(s('monitoring--selected-desk'), {hasText: 'new custom workspace'})).toBeVisible();
+        });
+
+        test('cancel creating custom workspace', async ({page}) => {
+            await restoreDatabaseSnapshot();
+            await page.goto('/#/workspace/monitoring');
+
+            await page.locator(s('monitoring--selected-desk')).click();
+            await page.locator(s('create-workspace')).click();
+            await expect(page.locator(s('workspace-modal', 'workspace-name'))).toBeVisible();
+            await expect(page.locator('button', {hasText: 'save'})).toBeDisabled();
```

Comment:
Scope your selector to the context. Go over all tests and fix the same issue. I won't comment on other occurences of this issue in this PR.